build/common/installer/scripts/tomlparser-geneva-config.rb

```diff
@@ -76,6 +77,11 @@ def populateSettingValuesFromConfigMap(parsedConfig)
                 end
               end
             end
+            enable_fbit_threading = parsedConfig[:integrations][:geneva_logs][:enable_threading].to_s
```

Comment:
Should we add this change in configmap as well?

Reply:
> Should we add this change in configmap as well?

> since this internal only feature hence not added to configmap.